SBLGNT/nodes/23-1john.xml

```diff
@@ -9374,7 +9374,7 @@
       <Node Id="1.60" Cat="O" Start="19" End="23" Rule="Np2O" Head="0" nodeId="620050010200051" Length="5">
         <Node Id="1.61" Cat="np" Start="19" End="23" Rule="AdvpNp" Head="1" nodeId="620050010200050" Length="5">
           <Node Id="1.62" Cat="advp" Start="19" End="19" Rule="Adv2Advp" Head="0" nodeId="620050010200011" Length="1">
-            <Node Id="1.63" Cat="adv" Start="19" End="19" StrongNumber="2532" English="also" UnicodeLemma="καί" Unicode="καὶ" Chinese="" Language="G" morphId="62005001020" encode="καὶ" nodeId="620050010200010" xml:id="n62005001020" ref="1JN 5:1!20" NormalizedForm="καί" UnicodeForm="καὶ" status="unmapped">καὶ</Node>
+            <Node Id="1.63" Cat="adv" Start="19" End="19" Lemma="kai/" StrongNumber="2532" English="also" Notes="" UnicodeLemma="καί" StrongNumberX="2532" Unicode="καὶ" Chinese="" Label="" Language="G" morphId="62005001020" encode="καὶ" RefIndex="" MorphId2="" RB="{}" NB="{}" AR="{}" BR="{}" nodeId="620050010200010" xml:id="n62005001020" ref="1JN 5:1!20" NormalizedForm="καί" UnicodeForm="καὶ" status="unmapped">καὶ</Node>
           </Node>
           <Node Id="1.64" Cat="np" Start="20" End="23" Rule="DetCL" Head="1" nodeId="620050010210040" Length="4">
             <Node Id="1.65" Cat="det" Start="20" End="20" StrongNumber="3588" English="the one" Case="Accusative" UnicodeLemma="ὁ" Unicode="τὸν" Chinese="（者）" Gender="Masculine" Number="Singular" Language="G" morphId="62005001021" encode="τὸν" nodeId="620050010210010" xml:id="n62005001021" ref="1JN 5:1!21" NormalizedForm="τόν" UnicodeForm="τὸν" FunctionalTag="T-ASM" Type="" FormalTag="T-ASM" Gloss="the [one]" LexDomain="092004" LN="92.24" status="mapped">τὸν</Node>
```

Comment:
@RickBrannan, looking good in general. Thank you! Are "unmapped" words the only ones that have the additional values as with the preceding sibling KAI in this verse?

Reply:
"Additional values" including things like `@Lemma`, which I see in ` <Node Id="1.63"...` but not  <Node Id="1.64"...`.

Reply:
@ryderwishart , Words in SBLGNT that I wasn't able to locate in N1904 using mapping data have the `status="unmapped` attribute. _Some_ of those `unmapped` items, however, did have data that could be migrated to `Ref`, `SubjRef`, and `Frame` attributes. So where possible I've migrated that data. 
In those `unmappped` nodes, the following attributes (if present) have been removed: 'Lemma', 'Label', 'Notes', 'StrongNumberX', 'RB', 'NB', 'AR', 'BR', 'MorphId2','RefIndex', 'Ref', 'SubjRef', 'Frame', 'RefEnglish', 'RefStrong', 'RefLemma','SubjRefEnglish', 'SubjRefStrong', 'SubjRefLemma'
What I'm unaware of in those cases is what required data, if any, need be added to the SBLGNT edition.

Reply:
There are `FormalTag` and `FunctionalTag` attributes in the N1904 that, for unmappable nodes, will not be available in SBLGNT. Should I try to get that data in (only `FormalTag`, I think, unsure how I'd get the `FunctionalTag` information) either based on the attributes that record morph fields or from Tauber's MorphGNT?